### PR TITLE
Makefile: fix variables

### PR DESCRIPTION
**Pull request by tbg**
No good deed goes unpunished: #17056 accidentally messed up the variables'
default values. For example, `TEST=".                              "`.
On the plus side, this makes our CI green much more often. But, I decided
to change it back.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,9 +54,6 @@ help: ## Print help for targets with comments.
 	@echo ""
 	@echo "Useful commands:"
 	@grep -Eh '^[a-zA-Z._-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "  $(shell tput setaf 6 2>/dev/null)%-30s$(shell tput sgr0 2>/dev/null) %s\n", $$1, $$2}'
-	@echo ""
```
Comment (mberhault) [inline Makefile]: why remove this? My understanding is that inline comments are the problem

**Review by mberhault — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (tschottdorf) [inline Makefile]: It doesn't print any variables any more. Let me merge this and send another PR for the bike shedding to avoid folks picking up the broken Makefile.
